[2023-10-26T04:59:15.654+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-10-26T04:58:59.045114+00:00 [queued]>
[2023-10-26T04:59:15.669+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-10-26T04:58:59.045114+00:00 [queued]>
[2023-10-26T04:59:15.670+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2023-10-26T04:59:15.688+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_data> on 2023-10-26 04:58:59.045114+00:00
[2023-10-26T04:59:15.699+0000] {standard_task_runner.py:57} INFO - Started process 268 to run task
[2023-10-26T04:59:15.702+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_data', 'manual__2023-10-26T04:58:59.045114+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/tugaskel8_dag.py', '--cfg-path', '/tmp/tmpddh55s1m']
[2023-10-26T04:59:15.705+0000] {standard_task_runner.py:85} INFO - Job 278: Subtask load_data
[2023-10-26T04:59:15.784+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_dag.load_data manual__2023-10-26T04:58:59.045114+00:00 [running]> on host d73c1ff20372
[2023-10-26T04:59:16.252+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kel8' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-26T04:58:59.045114+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-26T04:58:59.045114+00:00'
[2023-10-26T04:59:16.415+0000] {base.py:73} INFO - Using connection ID 'db_polusiudara' for task execution.
[2023-10-26T04:59:17.960+0000] {python.py:194} INFO - Done. Returned value was: Sukses Memasukkan Data
[2023-10-26T04:59:17.990+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_data, execution_date=20231026T045859, start_date=20231026T045915, end_date=20231026T045917
[2023-10-26T04:59:18.048+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-26T04:59:18.073+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
